enzyme/Enzyme/GradientUtils.cpp

```diff
@@ -2860,10 +2860,19 @@ Value *GradientUtils::invertPointerM(Value *const oval, IRBuilder<> &BuilderM,
   }
 
   if (isa<ConstantPointerNull>(oval)) {
+    if (width > 1) {
```

Comment:
Shouldn't apply chain rule work on a vector width of 1 regardless? Why have the if here?

Reply:
it should work without the if